dashboard/config/locales/scripts.en.yml

```diff
@@ -1437,3 +1437,8 @@ en:
           description_short: Learn to program droids, and create your own Star Wars game in a galaxy far, far away.
```

Comment:
if you don't want to commit this file, staging should generate the same content.  but this should be fine.